python/nav/web/crispyforms.py

```diff
@@ -195,3 +195,17 @@ class and template definitions.
         form_id=form_id,
         form_class=form_class,
     )
+
+
+class FormDiv:
+    """A class representing a div in a form layout.
+
+    :param fields: A list of fields to include in the div.
+    :param css_classes: Additional CSS classes to apply to the div.
+    """
+
+    def __init__(self, fields: list, css_classes: str = ''):
```

Comment:
`fields` should prob be optional? Since we have use cases where we just want a div with specific classes but not any content (`DetentionProfileForm`)